docs/deployment.md

```diff
@@ -0,0 +1,16 @@
+# Deployment
+
+GLAM has a 3-stage deployment:
+
+1. **Dev**: Deployments to the development server happen automatically for all commits that land on the master branch
+2. **Stage**: Deployments to the staging server happen automatically for tags of the form `m<integer>` or of the [CalVer](http://calver.org/) format `YYYY.MM.N`
```

Comment:
is it `YYYY.MM.N`? `N` isn't in the CalVer docs

Reply:
I can document that here. It's what we've done at mozilla on other projects. `N` is a zero-based counter for the number of tags of the month and does not relate to a day.